### PR TITLE
Release v6.6.1

### PR DESCRIPTION
**Pull request by tynandebold**
## Description

It's a patch release because there aren't any big, new features.

Here's what's going in:

#### Major features and improvements

- Skip all plugins while running Kedro-Viz using the `--ignore-plugins` option. (#1544)

#### Bug fixes and other changes

- Fix improper display of 'run-command' inside the metadata panel. (#1569)
- Replace semver with packaging. (#1578)
- Fix a bug in Kedro-Viz experiment tracking to ensure compatibility with `kedro-datasets>=1.7.1`. (#1600)

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [x] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

**Review by ravi-kumar-pilla — APPROVED** [review]
Looks good to me !! 💯

**Review by merelcht — APPROVED** [review]

**Comment by ravi-kumar-pilla** [comment]
@tynandebold should we mention anything about kedro-datasets version support here ? Like 
kedro-viz 6.6.1 supports kedro-datasets >=1.7.1, since experiment tracking is broken if someone uses kedro-viz 6.6.0 and kedro-datasets 1.7.1 ?

**Comment by tynandebold** [comment]
> @tynandebold should we mention anything about kedro-datasets version support here ? Like kedro-viz 6.6.1 supports kedro-datasets >=1.7.1, since experiment tracking is broken if someone uses kedro-viz 6.6.0 and kedro-datasets 1.7.1 ?

Mention it where exactly? We could add something to the documentation or the release notes. I don't think this blocks the release though.

**Review by vladimir-mck — APPROVED** [review]

**Comment by rashidakanchwala** [comment]
Merel's PR on fixing kedro-viz to work with kedro-datasets needs to be mentioned so I will add that since it's a bug at the moment. Also Merel's PR is not backward compatible, so it won't work with older kedro-datasets which means we might need to update our kedro-datasets dependency to >=1.7.1

File: RELEASE.md [hunk 1]
```diff
@@ -6,13 +6,14 @@ Please follow the established format:
...
-# Next Release
+# Release 6.6.1
 
 ## Major features and improvements
 
-- Skip all plugins while running Kedro Viz using the `--ignore-plugins` option. (#1544)
+- Skip all plugins while running Kedro-Viz using the `--ignore-plugins` option. (#1544)
 
 ## Bug fixes and other changes
+
```
Comment (rashidakanchwala) [inline RELEASE.md:16]: - Fix a bug in Kedro-viz Experiment Tracking to ensure compatibility with kedro-datasets > = 1.7.1

Comment (tynandebold) [reply RELEASE.md:16]: Good one. That's in now.

**Pull request merged**